index.js

```diff
@@ -1,59 +1,75 @@
 const arr = JSON.parse(localStorage.getItem('array')) || [];
 
-function add() {
-  const title = document.querySelector('#title');
-  const author = document.querySelector('#author');
-  const value1 = title.value.trim();
-  const value2 = author.value.trim();
-  if ((value1.length !== 0) && (value2 !== 0)) {
-    const book = {
-      title: value1,
-      author: value2,
-    };
-    arr.push(book);
-    localStorage.setItem('array', JSON.stringify(arr));
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
-}
-// this will remove a book
-function remove() {
-  const parent = document.querySelector('#list');
-  const button = document.querySelectorAll('#button');
-  const butto = Array.from(button).indexOf(this);
-  const reele = this.parentNode;
-  parent.removeChild(reele);
-  arr.splice(butto, 1);
-  localStorage.setItem('array', JSON.stringify(arr));
-}
-// This will display a book that is instantly added
-function display() {
-  const list = document.querySelector('#list');
-  const title = document.querySelector('#title');
-  const author = document.querySelector('#author');
-  const value1 = title.value.trim();
-  const value2 = title.value.trim();
-  if ((value1.length !== 0) && (value2 !== 0)) {
-    for (let i = arr.length - 1; i < arr.length; i += 1) {
+
+  display() {
+    const list = document.querySelector('#list');
+    const value1 = this.title.trim();
+    const value2 = this.author.trim();
+    const div = document.createElement('div');
+    div.classList.add('div');
+    div.innerHTML = `<div class="paradiv" <p>"${value1}" by "${value2}"</p></div><div class="buttondiv"><button id="button">Remove</button></div>`;
+    list.appendChild(div);
+    for (let i = 0; i < arr.length; i += 1) {
+      const button = document.querySelectorAll('#button');
+      button[i].addEventListener('click', Books.remove);
+    }
+    const change = document.querySelectorAll('.div');
+    for (let i = 0; i < arr.length; i += 1) {
+      if (i % 2 === 0) {
+        change[i].classList.add('background');
+      }
+    }
+  }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227172254-18bdf52b-842f-4800-ac0b-82c31c57b4d5.png)

**[OPTIONAL]**

- [ ] It would be great if you make the above message something like the below to make your readme more professional 👍

**Like below** 👇
Feel free to check the [issues](https://github.com/vikas924/Awesome-books/issues) page.

**Source** 👇

`Feel free to check the [issues](https://github.com/vikas924/Awesome-books/issues) page.`